Assets/BossRoom/Scripts/Shared/ScriptableObjects/Character.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using UnityEngine;
+
+namespace BossRoom
+{
+    /// <summary>
+    /// This ScriptableObject defines a Player Character for BossRoom. It defines its CharacterClass field for
+    /// associated game-specific properties, as well as its graphics representation.
+    /// </summary>
+    [CreateAssetMenu]
+    [Serializable]
+    public class Character : GuidScriptableObject
```

Comment:
Avatar

Reply:
Renamed, as well as other references to this.